src/components/auto-complete/auto-complete.st.css

```diff
@@ -0,0 +1,37 @@
+.root {
+    width: auto;
+    font-size: 16px;
+    position: relative;
+}
+
+.auto-complete-input {
+    border: 1px solid #adbac5;
+    font-family: ArialMT;
+    font-size: 16px;
+    height: 36px;
+    width: 100%;
+    padding-left: 5px;
+}
+
+.auto-complete-list {
+    -st-extends: List;
+    background: #FFFFFF;
+}
+
+.auto-complete-input:hover {
+    cursor: pointer;
+    border: 1px solid #5cb4ff;
+}
+
+.auto-complete-list {
+    width: 100%;
+}
+
+.caret {
+    cursor: pointer;
+    position:absolute;
+    top:50%;
+    left: 90%;
+    transform:translateY(-50%);
+    -webkit-transform:translateY(-50%);
```

Comment:
Why -webkit- prefix?

Reply:
for mobile

Reply:
which mobile requires -webkit-? Is it failing in the ci?

Reply:
Yes, we are running tests on android and it needs the webkit prefix to run